openshift/resolve-yamls.sh

```diff
@@ -52,4 +52,11 @@ function resolve_resources() {
 
     echo >>$resolved_file_name
   done
+
+  # handle additional images which are not build from ./cmd (images not in $CORE_IMAGES)
+  if [[ -n ${image_tag} ]];then
+      sed -i -r -e "s,github.com/tektoncd/pipeline/vendor/github.com/GoogleCloudPlatform/cloud-builders/gcs-fetcher/cmd/gcs-fetcher,${registry_prefix}-gcs-fetcher:${image_tag},g" $resolved_file_name
```

Comment:
this can be improved by using `variable` to store the long ... gcs-fetcher path

```
local gcf_fetcher=".............."
local image_url="....."
 [[ -n "$image_tag" ]] && image_url="$image_url:$image_tag"
sed -ire "s|$gcf_fetcher|$image_url|g" $resolved_files

```

Reply:
i see, it looks better, i ll update it

Reply:
i shall update it in the master branch, from there release next branch and all future branches will pick it up.